src/Domain/Creators/CylindricalBinaryCompactObject.cpp

```diff
@@ -801,7 +810,70 @@ Domain<3> CylindricalBinaryCompactObject::create_domain() const {
         CoordinateMaps::DiscreteRotation<3>(rotate_to_minus_x_axis));
   }
 
-  Domain<3> domain{std::move(coordinate_maps)};
+  // Excision spheres
+  std::unordered_map<std::string, ExcisionSphere<3>> excision_spheres{};
+
+  std::unordered_map<size_t, Direction<3>> abutting_directions_A;
+  size_t first_inner_sphere_block = 46;
+  if (include_inner_sphere_A_) {
+    for (size_t i = 0; i < 10; ++i) {
+      // LCOV_EXCL_START
+      abutting_directions_A.emplace(first_inner_sphere_block + i,
+                                    Direction<3>::lower_zeta());
+      // LCOV_EXCL_STOP
+    }
+    for (size_t i = 0; i < 4; ++i) {
+      // LCOV_EXCL_START
+      abutting_directions_A.emplace(first_inner_sphere_block + 10 + i,
+                                    Direction<3>::lower_xi());
+      // LCOV_EXCL_STOP
+    }
+    // Block numbers of sphereB might depend on whether there is an inner
+    // sphereA layer, so increment here to get that right.
+    first_inner_sphere_block += 14;
+  } else {
+    for (size_t i = 0; i < 5; ++i) {
+      abutting_directions_A.emplace(9 + i, Direction<3>::lower_zeta());
+      abutting_directions_A.emplace(27 + i, Direction<3>::lower_zeta());
+    }
+    for (size_t i = 0; i < 4; ++i) {
+      abutting_directions_A.emplace(14 + i, Direction<3>::lower_xi());
+    }
+  }
+  excision_spheres.emplace(
+      "ObjectAExcisionSphere",
+      ExcisionSphere<3>{radius_A_, rotate_from_z_to_x_axis(center_A_),
+                        abutting_directions_A});
+
+  std::unordered_map<size_t, Direction<3>> abutting_directions_B;
+  if (include_inner_sphere_B_) {
+    for (size_t i = 0; i < 10; ++i) {
+      // LCOV_EXCL_START
+      abutting_directions_B.emplace(first_inner_sphere_block + i,
+                                    Direction<3>::lower_zeta());
+      // LCOV_EXCL_STOP
+    }
+    for (size_t i = 0; i < 4; ++i) {
+      // LCOV_EXCL_START
+      abutting_directions_B.emplace(first_inner_sphere_block + 10 + i,
+                                    Direction<3>::lower_xi());
+      // LCOV_EXCL_STOP
```

Comment:
Unless this is an lcov bug (is it?), please don't add lcov excl statements to code that could reasonably be called in a test. The codecov CI test is not required to merge PRs so we can decide to ignore it.

Reply:
I think this code *is* called in the test.

Reply:
Yea this is weird. It worries me a bit because I haven't seen such an obvious lcov bug before. Well I have no change request here 🤷‍♂️